vllm/model_executor/guided_logits_processors.py

```diff
@@ -154,7 +185,7 @@ def __init__(self,
         super().__init__(regex_string, tokenizer)
 
 
-class CFGLogitsProcessor(BaseLogitsProcessor):
+class CFGLogitsProcessor(BaseGuidedLogitsProcessor):
 
     def __init__(self, cfg: str, tokenizer: PreTrainedTokenizerBase):
```

Comment:
Btw, `outlines.fsm.fsm.CFGFSM` is [deprecated](https://github.com/outlines-dev/outlines/blob/aa0a35ea728502216a6a33d5ba3d8d7e8c8c83d2/outlines/fsm/fsm.py#L55) and will be removed on 2024-06-01. I didn't find any particular segment of the code that heavily depends on `CDGFSM`, so I think switching to `outlines.fsm.guide.CFGGuide`

Reply:
If CFGGuide works that's great. I am aware of the deprecation.